nrf802154_sniffer/nrf802154_sniffer.py

```diff
@@ -69,6 +69,7 @@ class Nrf802154Sniffer(object):
 
     # Helpers for Wireshark argument parsing.
     CTRL_ARG_CHANNEL = 0
+    CTRL_ARG_LOGGER = 6
```

Comment:
Why 6? It it a special number? :)

Reply:
I kind of borrowed that value form @LuDuda implementation. Should have been `666` instead as this is a very cunning logger. :)